repairnator/repairnator-scanner/src/main/java/fr/inria/spirals/repairnator/scanner/Launcher.java

```diff
@@ -150,7 +169,14 @@ private void defineArgs() throws JSAPException {
 
         String googleSecretPath = this.arguments.getFile("googleSecretPath").getPath();
 
-        ProjectScanner scanner = new ProjectScanner(this.arguments.getInt("lookupHours"), launcherMode);
+        ProjectScanner scanner;
+        Date lookFromDate = this.arguments.getDate("lookFromDate");
+        Date lookToDate = this.arguments.getDate("lookToDate");
+        if (lookFromDate != null && lookToDate != null && lookFromDate.before(lookToDate)) {
+            scanner = new ProjectScanner(lookFromDate, lookToDate, launcherMode);
+        } else {
+            scanner = new ProjectScanner(this.arguments.getInt("lookupHours"), launcherMode);
```

Comment:
Actually we could even remove the constructor of Scanner with lookupHours and do the computing of the new date here. WDYT?